library/include/internal/hipsolver-compat.h

```diff
@@ -2,2674 +2,12 @@
  * Copyright (C) 2020-2024 Advanced Micro Devices, Inc.
  * ************************************************************************ */
 
-/*! \file
- *  \brief hipsolver-compat.h provides a compatibility API for users of
- *  cuSOLVER wishing to transition to hipSOLVER. Functions in this file
- *  have parameter lists matching those of cuSOLVER, but may suffer from
- *  performance issues when using the rocSOLVER backend. Switching to the
- *  use of functions from hipsolver-functions.h is recommended.
- */
+#pragma once
 
-#ifndef HIPSOLVER_COMPAT_H
-#define HIPSOLVER_COMPAT_H
```

Comment:
Let's keep `#pragma once` out of the public headers. We should continue to use the `HIPSOLVER_COMPAT_H` include guard.

Reply:
Understood; will do